### ISSUE TITLE
Generated tarball misses important files from test/

### ISSUE THREAD

**Issue by oerdnj** [icing/mod_md#40]
At least `Makefile.am` and `test.ini.in` are missing, maybe also others, so the tarball cannot be used to build the sources, as it fails to generate files in `test/` during configure phase:

```
config.status:1141: creating Makefile
config.status:1141: creating src/md_version.h
config.status:1141: creating src/Makefile
config.status:1127: error: cannot find input file: `test/Makefile.in'
```

**Comment by icing** [comment]
Sorry about that. Have added all files into the distribution in v0.8.2. Works on my MacOS box now.

**Comment by icing** [comment]
Fixed in v0.8.2. 
Patches added in v0.9.0.
